### PR TITLE
ACI Bridge Domain: Update docs

### PR DESCRIPTION
**Pull request by jmcgill298**

### CODE REVIEW

File: library/aci_bridge_domain.py [hunk 1]
```diff
@@ -32,6 +31,7 @@
     - Determines if the Bridge Domain should flood ARP traffic.
     - The APIC defaults new Bridge Domains to "no".
     choices: [ no, yes ]
```
Comment (dagwieers) [inline library/aci_bridge_domain.py]: If  you really want a boolean, this should actually be:
```yaml
    type: bool
    default: 'no'
```
Which is to ensure that the documentation prints 'no' and not False IIUC.

File: library/aci_bridge_domain.py [hunk 2]
```diff
@@ -41,72 +41,82 @@
...
+    default: ethernet
   description:
     description:
     - Description for the Bridge Domain.
   enable_multicast:
     description:
     - Determines if PIM is enabled
-    - The APIC defaults new Bridge Domains to disabled.
+    - The APIC defaults new Bridge Domains to "no".
     choices: [ no, yes ]
```
Comment (dagwieers) [inline library/aci_bridge_domain.py]: If  you really want a boolean, this should actually be:
```yaml
    type: bool
    default: 'no'
```
Which is to ensure that the documentation prints 'no' and not False IIUC.

File: library/aci_bridge_domain.py [hunk 3]
```diff
@@ -41,72 +41,82 @@
...
+    default: no
   enable_routing:
     description:
     - Determines if IP forwarding should be allowed.
-    - The APIC defaults new Bridge Domains to IP forwarding enabled.
+    - The APIC defaults new Bridge Domains to "yes".
     choices: [ no, yes ]
```
Comment (dagwieers) [inline library/aci_bridge_domain.py]: If  you really want a boolean, this should actually be:
```yaml
    type: bool
    default: 'no'
```
Which is to ensure that the documentation prints 'no' and not False IIUC.

File: library/aci_bridge_domain.py [hunk 4]
```diff
@@ -41,72 +41,82 @@
...
+    default: yes
   endpoint_clear:
     description:
-    - Clears all End Points in all Leaves when enabled.
-    - The APIC defaults new Bridge Domains to disabled.
+    - Clears all End Points in all Leaves when "yes".
+    - The APIC defaults new Bridge Domains to "no".
     - The value is not reset to disabled once End Points have been cleared; that requires a second task.
     choices: [ no, yes ]
```
Comment (dagwieers) [inline library/aci_bridge_domain.py]: If  you really want a boolean, this should actually be:
```yaml
    type: bool
    default: 'no'
```
Which is to ensure that the documentation prints 'no' and not False IIUC.

File: library/aci_bridge_domain.py (hunk 1 above)
Comment (jmcgill298) [inline library/aci_bridge_domain.py]: @dagwieers the APIC usually uses 'enabled' and 'disabled', but this particular object was implemented with 'yes' and 'no'. These are strings and not bools. If we want to move all the modules to use bools, then we can update with all the rest.

Comment (jmcgill298) [inline library/aci_bridge_domain.py]: and this is why `type: str` should be valid 😆

**Comment by dagwieers** [comment]
Why was this closed ? You can always fix an existing PR.

**Pull request closed without merging**